common/lumens.js

```diff
@@ -1,10 +1,10 @@
 // This file contains functions used both in backend and frontend code.
 // Will be helpful to build distribution stats API.
-import axios from "axios";
-import BigNumber from "bignumber.js";
-import map from "lodash/map";
-import reduce from "lodash/reduce";
-import find from "lodash/find";
+const axios = require("axios");
+const BigNumber = require("bignumber.js");
+const map = require("lodash/map");
+const reduce = require("lodash/reduce");
+const find = require("lodash/find");
```

Comment:
needed to remove babel on the backend because of a [bug with sequelize](https://github.com/sequelize/sequelize/issues/7840#issuecomment-315576088) (annoyingly since sequelize will be removed soon). So changed common/lumens.js to use commonJS common/require instead of import/export which works for both the frontend and backend

Reply:
Do we need to keep this file in JavaScript?

Reply:
yup, at least until the frontend uses typescript because it's importing this file
eg https://github.com/stellar/dashboard/blob/6ec7ac5bebc0d12f22a848a426ffdf1964f06e27/frontend/components/LumensCirculating.js#L5